### PR TITLE
Fix method convertLiterToGallon #301

### LINKED ISSUE #301: Fix method convertLiterToGallon
There is currently a method called `convertLiterToGallon` that lives in `/lib/convertLiterToGallon.js`.
It is incomplete and needs to be fixed!

### Acceptance Criteria:
1. Running `yarn test convertLiterToGallon` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by damjanh**
Closes #301 

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: lib/convertLiterToGallon.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
- * @param {number} liter 
- * @return returned value in gallon  
+ * @param {number} liter
+ * @return returned value in gallon
  */
 
 function convertLiterToGallon(liter) {
-  return;
+  if (typeof liter != "number") {
+    throw new Error("Paramter must not be a number.");
```
Comment (BlakeGuilloud) [inline lib/convertLiterToGallon.js]: I think this should read: "Parameter must be a number."

**Comment by damjanh** [comment]
Great catch @BlakeGuilloud. Thx.

**Comment by damjanh** [comment]
New issue created for the new skeleton method.
